modules/admin_manual/pages/enterprise/external_storage/enterprise_only_auth.adoc

```diff
@@ -1,45 +1,205 @@
-* SFTP
-* SMB/CIFS
-* WebDAV
-* Windows Network Drive
+The following table shows which authentication option is available for which edition of ownCloud. 
+
+[width="70%",cols="30%,^20%,^20%",options="header"]
+|===
+| Authentication Method
+| Free Edition
+| Enterprise Edition
+
+| Username and password
+| x
+| x
+
+| Log-in credentials, save in session
+| x
+| x
+
+| Log-in credentials, save in database
+| 
+| x
+
+| User entered, store in database
+| 
+| x
+
+| Credentials hardcoded in config file
+| 
+| x
+
+| Global credentials
+| 
+| x
+
+| RSA public key *
+| x
+| x
+|===
+
+(*) Note that `RSA public key` is only available for SFTP.
+
+== Storage Implementations
+
+The following table shows which storage implementation can use the above authentication options.
+Note that this table does not cover other storage implementations like Google Drive or One Drive ect, as those use other authentication backends.
+
+[width="70%",cols="30%,^20%,^20%",options="header"]
+|===
+| Storage implementation
+| Free Edition
+| Enterprise Edition
+
+| FTP
+| x
+| x
+
+| SFTP
+| x
+| x
+
+| ownCloud
+| x
+| x
+
+| SMB/CIFS
+| x
+| x
+
+| WebDAV
+| x
+| x
+
+| Windows Network Drive
+| 
+| x
+|===
+
+Use the dropdown selector to choose the authentication backend when you create a new external mount.
+
+image:enterprise/external_storage/enterprise-only-auth/auth_backends-1.png[Authentication dropdown selector.]
+
+== Authentication Option Details
 
 Username and password::
-  This is the default; a login entered by the admin when the external
-  mount is created. The login is stored in the database, which allows
-  sharing, and background jobs, such as file scanning, to operate.
+  This is the default; a login entered by the admin when the external mount is created. The login is stored in the database, which allows sharing and background jobs, such as file scanning, to operate.
 Log-in credentials, save in session::
-  Credentials are only stored in the session and not captured in the
-  database. Files cannot be shared, as credentials are not stored.
+  Credentials are only stored in the session and not captured in the database. Files cannot be shared, as credentials are not stored.
 Log-in credentials, save in database::
   Credentials are stored in the database, and files can be shared.
 User entered, store in database::
-  Users provide their own login credentials, rather than using
-  admin-supplied credentials. User credentials are stored in the
-  database, and files can be shared.
+  Users provide their own login credentials, rather than using admin-supplied credentials. User credentials are stored in the database, and files can be shared.
+Credentials hardcoded in config file::
+  Used when all the users will share the same password and the password is not known to the users like for a shared space. The user id is always taken from the session. This authentication option can also be chosen when using oAuth or OpenIDConnect etc. Files can be shared.
```

Comment:
For details see below, section "Notes for Credentials Hardcoded in Config File"

Reply:
fixed, link added